bin/server.js

```diff
@@ -40,4 +42,24 @@ DB.connect(config)
     server.on('mem', function (stats) {
       logger.info('mem', stats)
     })
+    // Publish notifications via simple background loop.
+    if (config.notifications.publishUrl) {
+      var sleepTime
```

Comment:
It...huh.  Yes, yes it could be and should be.  I think this higher-level scoping is leftover from a previous factoring of the code.